server/portal/apps/accounts/views.py

```diff
@@ -19,6 +21,28 @@
 logger = logging.getLogger(__name__)
 
 
+class LogoutView(DjangoLogoutView):
+    def dispatch(self, request, *args, **kwargs):
+        token = str(request.user.tapis_oauth.access_token)
```

Comment:
Already a string I think?
```suggestion
        token = request.user.tapis_oauth.access_token
```